include/wimlib/win32.h

```diff
@@ -38,7 +38,7 @@ int
 fsync(int fd);
 
 tchar *
-realpath(const tchar *path, tchar *resolved_path);
+wimlib_realpath(const tchar *path, tchar *resolved_path);
```

Comment:
Can you drop the `resolved_path` argument and just use `tchar *wimlib_realpath(const tchar *path)`?  Support for non-null `resolved_path` is not required, so this would be simpler.